src/main/scala/com/lynbrookrobotics/seventeen/CoreRobot.scala

```diff
@@ -392,9 +392,9 @@ class CoreRobot(configFileValue: Signal[String], updateConfigFile: String => Uni
 
 object CoreRobot {
   implicit class ToTimeSeriesNumeric[T](val stream: Stream[T]) extends AnyVal {
-    def toTimeSeriesNumeric(name: String)(implicit ev: T => Double): TimeSeriesNumeric = {
+    def toTimeSeriesNumeric(name: String)(whatUnits: String)(implicit ev: T => Double): TimeSeriesNumeric = {
```

Comment:
I think `whatUnits` would be better suited as just another parameter instead of being curried, since it's just another configuration value.